### ISSUE TITLE
Better installation command

### ISSUE THREAD

**Issue by wagenet** [paddle8/ember-popup-menu#12]
Won't `ember install ember-popup-menu` do the same as the two liner you have right now?

**Comment by tim-evans** [comment]
Yes, it definitely will. This is a fairly old addon that I'm trying to give some more love to

**Comment by wagenet** [comment]
Got it :)

**Comment by tim-evans** [comment]
(See https://github.com/tim-evans/ember-pop-over for the WIP)
